common/client/models.go

```diff
@@ -0,0 +1,17 @@
+package client
+
+type SendTxReturnCode int
```

Comment:
Do these have a canonical string form? That would be required for https://smartcontract-it.atlassian.net/browse/BCF-1793